pkg/service/user.go

```diff
@@ -52,6 +52,8 @@ type UserService interface {
 	UpdateIdpUserGroupPolicy(context.Context, string, string, string) error
 	// Generate recovery link for users
 	ForgotPassword(context.Context, *userrpcv3.UserForgotPasswordRequest) (*userrpcv3.UserForgotPasswordResponse, error)
+	// Generate auditLog event
+	AuditLogWebhook(context.Context, *userrpcv3.AuditWebhook) (*userrpcv3.AuditWebhook, error)
```

Comment:
name it better at service layer - probably something like CreateLoginAuditLog

Reply:
made the change